frontend/src/layout/lemonade/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -23,6 +23,7 @@ function Breadcrumb({ breadcrumb }: { breadcrumb: IBreadcrumb }): JSX.Element {
         >
             {breadcrumb.symbol}
             {breadcrumb.name}
+            {breadcrumb.popup && <CaretDownOutlined style={{ color: 'var(--muted-alt)', marginLeft: 4 }} />}
```

Comment:
Let's use `IconArrowDropDown` (from `icons.tsx` instead of AntD), since that's the one used in Figma designs.

Reply:
Perfect!